public/js/controllers/systems.graphs.controller.js

```diff
@@ -0,0 +1,107 @@
+import { AreaChart } from "../libs/area.chart.js";
+
+export class SystemsGraphsController {
+    static updateGraphs(data) {
+        // Clear previous graphs before redrawing
+        d3.selectAll("svg").remove();
+
+/*                         DISK UTILIZATION GRAPH
+/*      ==================================================================
```

Comment:
I don't think this huge block comment is necessary now that's there not much code below, and that the variables have sensible names. I think you can get away with // Draw disk utilization graph or no comment at all. If you think you need a large block comment like this, it's a good sign that the code below should really be in its own function

Reply:
Updated comments i.e., // Draw disk utilization graph. thanks